services/datamanager/builtin/noop_client_conn_with_connectivity_test.go

```diff
@@ -0,0 +1,66 @@
+package builtin
```

Comment:
These are used to test how datamanger behaves when the connection with app.viam.com is healthy vs unhealthy vs doesn't implement the methods we expect to be there to determine health (which would be a bug in the cloud service).